index.bs

```diff
@@ -2858,21 +2858,27 @@ the inputs to the algorithm needed to compute the set.
 An effective overload set is used, among other things, to determine whether there are ambiguities in the
 overloaded operations, constructors and callers specified on an interface.
 
-The elements of an effective overload set are tuples of the form
-&lt;|callable|, |type list|, |optionality list|&gt;.  If the effective overload
-set is for regular operations, static operations or legacy callers, then |callable| is an operation;
-if it is for constructors or named constructors, then |callable| is an
-extended attribute; and if it is for callback functions, then |callable|
-is the callback function itself.  In all cases, |type list| is a list
-of IDL types, and |optionality list| is a list of three possible <dfn id="dfn-optionality-value" export>optionality values</dfn> –
-“required”, “optional” or “variadic” – indicating whether
-the argument at a given index was declared as being [=optional argument|optional=]
-or corresponds to a [=variadic=] argument.
-Each tuple represents an allowable invocation of the operation,
+The [=set/items=] of an [=effective overload set=] are [=tuples=] of the form
+([=effective overload set tuple/callable=], [=type list=], [=optionality list=])
+whose [=tuple/items=] are described below:
+
+*   The <dfn for="effective overload set tuple" export>callable</dfn> is an operation
```

Comment:
Thanks for catching that; Edit gone wrong (it used to say "The callable tuple item" which was overly verbose, imho).

Would you go with:

```
*   <dfn for="effective overload set tuple" export>Callable</dfn> is an operation
    if the effective overload set is for regular operations, static operations or legacy callers;
    it is an extended attribute if the effective overload set is for constructors or named constructors;
    and it is the callback function itself if the effective overload set is for callback functions.
*   <dfn export>Type list</dfn> is a [=list=] of IDL types.
*   <dfn export>Optionality list</dfn> is a [=list=] of
    three possible <dfn id="dfn-optionality-value" export>optionality values</dfn>
    – "required", "optional" or "variadic" –
    indicating whether the argument at a given index was
    declared as being [=optional argument|optional=]
    or corresponds to a [=variadic=] argument.
```

or a definition list:

```
 :  <dfn for="effective overload set tuple" export>Callable</dfn>
 :: If the effective overload set is for regular operations, static operations or legacy callers,
    then callable is an operation;
    If the effective overload set is for constructors or named constructors,
    then callable is an extended attribute;
    and if the effective overload set is for callback functions,
    then callable is the callback function itself.
 :  <dfn export>Type list</dfn>
 :: A [=list=] of IDL types.
 :  <dfn export>Optionality list</dfn>
 :: A [=list=] of three possible <dfn id="dfn-optionality-value" export>optionality values</dfn>
    – "required", "optional" or "variadic" –
    indicating whether the argument at a given index was
    declared as being [=optional argument|optional=]
    or corresponds to a [=variadic=] argument.
```